site/en/blog/signed-exchanges-desktop/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Signed Exchanges launched for Desktop sites 
+description: >
+  Announcing Signed Exchanges Support for Chromium Browser on Desktop along with other recent feature launches.
+date: 2022-07-08
+authors:
+  - twifkak
+---
+
+[Signed Exchanges (SXG)](https://web.dev/signed-exchanges/) is a delivery mechanism that can help speed up your site and improve [Largest Contentful Paint (LCP)](https://web.dev/lcp/) by enabling privacy-preserving cross-origin prefetch. At [I/O 2022](https://www.youtube.com/watch?v=bxc-Gc8KmF4), the SXG working group at Google previewed plans to add new SXG features and support site navigations on additional surfaces. Today, we are announcing the first set of these features, with more to come over the next few months:
+
+- Support for Desktop site navigations 
+- Google Search support for SXG from Top Stories & News 
+- Support for server side personalization 
+
+Read on to learn more about these features and how you can leverage them to improve the user experience on your site.
+
+## Support for Desktop Navigations
+
+With this release, Signed Exchanges will now be available for Desktop cross-origin navigations on Chromium browsers. This builds on the existing support for Android mobile and means that now, you can use SXG to also cache and prefetch the desktop version of your site. 
+
+We estimate<sup><a href="#footnote-1">1</sup> that this release will, on average, double SXG's coverage of your site's visits, enabling improved loading and performance for more users. Today on mobile, this speed improvement is between 300 and 400 ms of LCP reduction on average for navigations from Google Search.
```

Comment:
When reviewing this in rich diff mode, the footnote link seems to apply to all of the text in the paragraph after the superscript